client/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015", "stage-0", "react"]
+}
```

Comment:
Keep on the lookout for these red symbols. Means you're missing a newline at the end of the file.